core/src/main/java/io/questdb/cairo/frm/file/ContiguousFileIndexedFrameColumn.java

```diff
@@ -59,20 +57,22 @@ public void append(long offset, FrameColumn sourceColumn, long sourceLo, long so
                     indexWriter.add(TableUtils.toIndexKey(Unsafe.getUnsafe().getInt(mappedAddress + (i << shl))), offset + i);
                 }
                 indexWriter.setMaxValue(offset + size - 1);
+                indexWriter.commit();
```

Comment:
nit: commit mode is ignored

Reply:
`BitmapIndexWriter` fetches the commit mode from `CairoConfiguration`, so it respects the configured commit mode.

Reply:
Andrei is right